main.css

```diff
@@ -0,0 +1,69 @@
+*{

```

Comment:
Для удобности чтения лучше использовать следующий формат
```suggestion
* {
```